content/_snippets/no-cli-syntax.md

```diff
@@ -0,0 +1 @@
+**Note:** There is no commandline syntax for `ledger_data`. You can use the [json method][] to access this method from the commandline instead.
```

Comment:
The snippet needs to be generic to all the relevant methods, not just `ledger_data`.

```suggestion
**Note:** There is no commandline syntax for this method. You can use the [json method][] to access this method from the commandline instead.
```